### DISCUSSION TITLE
Add a close button to  "Input with button".

### DISCUSSION THREAD

**Discussion by JimLovesPineapples** [mantinedev/mantine#540] [Feedback]
Hey, I was going through the notifications section and saw it had no close button or a timeout. 
Of course when in dev you can put one but I think It would be better if the docs had one since I need to refresh to make it go away.
Also loving the hooks  and can't wait to try them out on my next project !

**Comment by rtivital** [comment]
You can click send button to make it go away (it does not actually send your email anywhere), that's fair though, I think we can add a close button to now confuse anyone, feel free to submit a PR with a fix if you want to contribute

**Reply by jerebtw** [reply]
Should it auto close or allow close? @rtivital 

Because I think a close button doesn't look that good.

![image](https://user-images.githubusercontent.com/49494752/146969626-b05ac210-4568-4a46-ac43-67ab78ed3899.png)

**Reply by rtivital** [reply]
It should auto close
